lua_examples/pcm/play_file.lua

```diff
@@ -0,0 +1,40 @@
+-- ****************************************************************************
+-- Play file with pcm module.
+--
+-- Upload jump_8k.u8 to spiffs before running this script.
+--
+-- ****************************************************************************
+
+
+function cb_drained(d)
+  print("drained "..node.heap())
+
+  file.seek("set", 0)
+  -- uncomment the following line for continuous playback
+  --d:play(pcm.RATE_8K)
+end
+
+function cb_stopped(d)
+  print("playback stopped")
+  file.seek("set", 0)
+end
+
+function cb_paused(d)
+  print("playback paused")
+end
+
+file.open("jump_8k.u8", "r")
+
+drv = pcm.new(pcm.SD, 1)
```

Comment:
I am pretty new at all of this, and having audio come out of the nodemcu is kina awesome. When I didn't have sigma-delta installed, I got an error when trying to access, pcm.SD. After adding sigma-delta to my firmware, everything started working. Is there a way to avoid calling pcm.SD or can you please post a sample of how to run without adding sigma-delta to the firmware?